titus-api/src/main/java/com/netflix/titus/api/appscale/model/PolicyConfiguration.java

```diff
@@ -104,6 +106,7 @@ public Builder withTargetTrackingPolicy(TargetTrackingPolicy targetTrackingPolic
         }
 
         public PolicyConfiguration build() {
+            Preconditions.checkNotNull(alarmConfiguration, "AlarmConfiguration is null");
```

Comment:
Does this result in a proper INVALID_ARGUMENT return type to users? It would be good to know that this doesn't give some obscure error.

Reply:
Also, are we still planning on adding Java Bean validation or just rely on this?

Reply:
This was a quick fix to prevent persisting an invalid value and cause GetAllScalingPolicies API to fail.
Since we have temporary remediation steps, I can look into adding some more validation and tests to confirm the new behavior.